app/models/card.rb

```diff
@@ -3,6 +3,15 @@ class Card < ActiveRecord::Base
   validates :original_text, :translated_text, :add_review_date, presence: true
   validate :words_cannot_be_equal
 
+  scope :random_for_review, -> { where('review_date <= ?', Time.now)
```

Comment:
Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.